EXADGConfig.cmake.in

```diff
@@ -0,0 +1,15 @@
+set(EXADG_VERSION @VERSION@)
```

Comment:
I think we use capital letters for all cmake commands in exadg.

Reply:
You are right. I fixed this :)